### DISCUSSION TITLE
What does it mean when the `text` for marks is a table?

### DISCUSSION THREAD

**Discussion by mawkler** [petertriho/nvim-scrollbar#70] [Q&A]
What does it mean when the `text` field of a mark is a table? For instance, by default the `Error` mark looks like this:

```lua
{
    text = { "-", "=" },
    -- ...
},
```

I'm guessing that `text` is the symbol that is shown for the mark in the scrollbar, but what does it mean when there are more than one? I couldn't find any documentation that explains it.

**Comment by petertriho** [comment]
Yep, text is the symbol shown for the mark in the scrollbar. When there are more than one, it'll move to the next text value (if available)

**Reply by mawkler** [reply]
I see! Perhaps there should be the possibility of adding a third text value which could be `≡` by default, for when there are more than two marks?
